contrib/opencensus-ext-flask/tests/test_flask_middleware.py

```diff
@@ -278,13 +279,37 @@ def test__after_request_sampled(self):
         flask_trace_id = '00-{}-{}-00'.format(trace_id, span_id)
 
         app = self.create_app()
-        flask_middleware.FlaskMiddleware(app=app)
+        flask_middleware.FlaskMiddleware(
+            app=app,
+            sampler=samplers.AlwaysOnSampler()
```

Comment:
Note: Adding the `AlwaysOnSampler` made sure the request was always sampled, which did not seem to be the case before

Reply:
@reyang don't know if we should do this, this was changed by @c24t [here](https://github.com/census-instrumentation/opencensus-python/pull/639). Waiting for his opinion on this.

Reply:
See comments [here](https://github.com/census-instrumentation/opencensus-python/pull/639#issuecomment-488202973)

Reply:
@victoraugustolls I’m aligned with @c24t comments regarding sampling on a production app, however I’m not sure how that’s a problem in tests?

The thing is, until I did this the tests wouldn’t pass because the default sampler has such a low sampling probability that the test span would actually never be created. I do think in the case of unit tests, we’d rather have deterministic execution than chance-driven tests :slightly_smiling_face:

Reply:
Even being tests, I would change for 100% Probability Sampler because:

> AlwaysOnSampler is redundant though, and AFAIK just exists for completeness.

So who knows, in the future it may change, although I don't think it will!

Reply:
> @victoraugustolls I’m aligned with @c24t comments regarding sampling on a production app, however I’m not sure how that’s a problem in tests?
> 
> The thing is, until I did this the tests wouldn’t pass because the default sampler has such a low sampling probability that the test span would actually never be created. I do think in the case of unit tests, we’d rather have deterministic execution than chance-driven tests slightly_smiling_face

Wow, what a luck :D, never happened to me!

Reply:
+1 on having more deterministic execution in test case.